src/lib/button/_button-theme.scss

```diff
@@ -107,7 +107,7 @@
 }
 
 @mixin mat-button-typography($config) {
-  .mat-button, .mat-raised-button, .mat-icon-button {
+  .mat-button, .mat-raised-button, .mat-icon-button, .mat-fab, .mat-mini-fab {
```

Comment:
The spec doesn't really have fabs with text, if I remember correctly, but it shouldn't be an issue to keep it anyway.